opportunity/examples/eusi/OpportunityCollectionResponseEusi.json

```diff
@@ -0,0 +1,104 @@
+{
+	"type": "FeatureCollection",
+	"features": [
+		{
+			"type": "Feature",
+			"geometry": {
+				"type": "Polygon",
+				"coordinates": [
+					[
+						[
+							13.34140753116992,
+							52.5427526175211
+						],
+						[
+							13.34140753116992,
+							52.4722618327375
+						],
+						[
+							13.500875567587116,
+							52.4722618327375
+						],
+						[
+							13.500875567587116,
+							52.5427526175211
+						],
+						[
+							13.34140753116992,
+							52.5427526175211
+						]
+					]
+				]
+			},
+			"properties": {
+				"id": "uuid-identifier-goes-here",
+				"product_id": "maxar_opt",
+				"collectionLikelihood": 0.8,
+        "unit": 50,
+        "unit_description": "km2",
+        "price_per_unit": 5.00,
+        "currency": "EUR",
+        "price_total_net": "250.00",
```

Comment:
got some mixed cases here, snake and camel, consider aligning on one or the other